product-plane-services/policy-server/src/main/java/org/opendatamesh/platform/pp/policy/server/utils/SpELUtils.java

```diff
@@ -0,0 +1,69 @@
+package org.opendatamesh.platform.pp.policy.server.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.opendatamesh.platform.core.commons.servers.exceptions.BadRequestException;
+import org.opendatamesh.platform.pp.policy.api.resources.PolicyEvaluationRequestResource.EventType;
+import org.opendatamesh.platform.pp.policy.api.resources.exceptions.PolicyApiStandardErrors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.expression.EvaluationException;
+import org.springframework.expression.Expression;
+import org.springframework.expression.ExpressionParser;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+
+public final class SpELUtils {
+
+    private static final ExpressionParser SPEL_PARSER = new SpelExpressionParser();
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SpELUtils.class);
+
+    public static Boolean evaluateSpELExpression(JsonNode inputObject, String spelExpression, EventType eventType) {
```

Comment:
qui puoi ritornare boolean direttamente

Reply:
ps. magari rinominala in eventObjectMatchesSpelExpr, perchè cosi non si capisce benissimo cosa fa la funzione da fuori